autox/k8sclient.go

```diff
@@ -17,10 +17,6 @@ import (
 // Client is a global variable. Before use, it must be assigned and initalized
 var Client kubeClient
```

Comment:
May be `k8sClient` is a better name than `Client`. (note caps; also, the type of client is obvious from the name).